### PR TITLE
Create 'display registered address' page

### PR DESCRIPTION
**Pull request by kiranprakashdigital**
### Description of Work carried through

- Added confirm company address page
- Added the same for group member
- Changed the partnership selection to navigate to Grs
- Populating partnership details on grs call back

Brief description of what/why/how.

#### Check list 
 - [x] `./precheck` was executed (Integration/Component/Unit tests)
 - [x] `sbt scalafmt test:scalafmt` was executed
 - [x] User Acceptance Tests (UAT) were run locally and have passed.
 - [x] No Accessibility errors/warnings reported by Wave
Screenshot_20211220_162510

### CODE REVIEW

File: conf/messages.en [hunk 1]
```diff
@@ -456,4 +456,6 @@ group.member.already.registered.title = You cannot add {0} to the group
 group.member.already.registered.detail1 = {0} has already registered for PPT, so cannot be added to the group.
 group.member.already.registered.detail2 = {0} will need to de-register for PPT before you can add them to the group.
 
-amendRegistration.organisationDetails.title = Your PPT registration
\ No newline at end of file
+amendRegistration.organisationDetails.title = Your PPT registration
```
Comment (paulblakehmrc) [inline conf/messages.en:459]: Is this from Ryan's PR already on main?

**Review by paulblakehmrc — APPROVED** [review]
LGTM - a couple of minor points

File: conf/messages.en (hunk 1 above)
Comment (RyanKellyHMRC) [reply conf/messages.en:459]: It's just a newline at the end of a line which was added by my PR. No probs.

**Review by RyanKellyHMRC — CHANGES_REQUESTED** [review]
Just a couple of minor requests.

File: test/spec/PptTestData.scala [hunk 2]
```diff
@@ -90,7 +81,7 @@ trait PptTestData extends RegistrationBuilder with MockAuthAction {
 
   val request: Request[AnyContent] = FakeRequest().withCSRFToken
 
-  protected val testCompanyName   = "Example Limited"
+  protected val testCompanyName   = "Example Limite"
```
[the remark this answers is not in the document]
Comment (kiranprakashdigital) [reply test/spec/PptTestData.scala:84]: I was testing something, forgot to revert.

File: app/uk/gov/hmrc/plasticpackagingtax/registration/controllers/group/ConfirmGroupMemberBusinessAddressController.scala [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+  authenticate: AuthAction,
+  journeyAction: JourneyAction,
+  override val registrationConnector: RegistrationConnector,
+  mcc: MessagesControllerComponents,
+  page: confirm_business_address
+)(implicit ec: ExecutionContext)
+    extends FrontendController(mcc) with Cacheable with I18nSupport {
+
+  def displayPage(): Action[AnyContent] =
+    (authenticate andThen journeyAction).async { implicit request =>
```
[the remark this answers is not in the document]
Comment (kiranprakashdigital) [reply app/uk/gov/hmrc/plasticpackagingtax/registration/controllers/group/ConfirmGroupMemberBusinessAddressController.scala:48]: Done

File: app/uk/gov/hmrc/plasticpackagingtax/registration/controllers/organisation/ConfirmBusinessAddressController.scala [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+  authenticate: AuthAction,
+  journeyAction: JourneyAction,
+  override val registrationConnector: RegistrationConnector,
+  mcc: MessagesControllerComponents,
+  page: confirm_business_address
+)(implicit ec: ExecutionContext)
+    extends FrontendController(mcc) with Cacheable with I18nSupport {
+
+  def displayPage(): Action[AnyContent] =
+    (authenticate andThen journeyAction).async { implicit request =>
```
[the remark this answers is not in the document]
Comment (kiranprakashdigital) [reply app/uk/gov/hmrc/plasticpackagingtax/registration/controllers/organisation/ConfirmBusinessAddressController.scala:50]: Done

File: conf/app.routes [hunk 5]
```diff
@@ -158,3 +158,7 @@ POST       /amend-phone-number      uk.gov.hmrc.plasticpackagingtax.registration
 
 GET        /amend-address      uk.gov.hmrc.plasticpackagingtax.registration.controllers.amendment.AmendContactDetailsController.address()
 POST       /amend-address      uk.gov.hmrc.plasticpackagingtax.registration.controllers.amendment.AmendContactDetailsController.updateAddress()
+
+GET        /confirm-address      uk.gov.hmrc.plasticpackagingtax.registration.controllers.organisation.ConfirmBusinessAddressController.displayPage()
+
+GET        /group-confirm-address      uk.gov.hmrc.plasticpackagingtax.registration.controllers.group.ConfirmGroupMemberBusinessAddressController.displayPage()
```
[the remark this answers is not in the document]
Comment (kiranprakashdigital) [reply conf/app.routes:164]: Done

File: test/spec/PptTestData.scala (hunk 2 above)
[the remark this answers is not in the document]
Comment (kiranprakashdigital) [reply test/spec/PptTestData.scala:84]: Done

File: test/uk/gov/hmrc/plasticpackagingtax/registration/controllers/group/GroupMemberGrsControllerSpec.scala [hunk 6]
```diff
@@ -92,46 +105,74 @@ class GroupMemberGrsControllerSpec extends ControllerSpec {
...
+              await(simulateLimitedCompanyCallback(registrationWithSelectedGroupMember(orgType)))
+          }
 
           groupMemberSize(getLastSavedRegistration) mustBe 1
 
           val memberDetails: GroupMember = getLastSavedRegistration.groupDetail.get.members.last
 
           memberDetails.organisationDetails.get.organisationType mustBe orgType.toString
-          memberDetails.organisationDetails.get.organisationName mustBe incorporationDetails.companyName
-          memberDetails.addressDetails mustBe incorporationDetails.companyAddress.toPptAddress
```
[the remark this answers is not in the document]
Comment (kiranprakashdigital) [reply test/uk/gov/hmrc/plasticpackagingtax/registration/controllers/group/GroupMemberGrsControllerSpec.scala:113]: I was trying to group the changes for partnership and remaining work with less code.. at that point I might have removed and not put it back. Now have put it back. Done

**Review by RyanKellyHMRC — APPROVED** [review]
All good. Thanks.

**Pull request merged**